### ISSUE TITLE
R turn off CRAN incoming check by default.

### ISSUE THREAD

**Issue by jimhester** [travis-ci/travis-ci#3337]
Would it be possible to set _R_CHECK_CRAN_INCOMING_=false as an environment variable in these builds by default? Doing so turns off the checking CRAN incoming feasibility check which is unnecessary except for the actual CRAN submission, increases the load on the CRAN servers handling the extra requests and takes ~10 seconds extra time on average.

This option was defaulted to FALSE previously in r-travis https://github.com/craigcitro/r-travis/blame/bb0483842206095b0c8ad6eddadefc77242c3152/scripts/travis-tool.sh#L277

cc @craigcitro

**Comment by craigcitro** [comment]
you should be able to add

    - env
      global:
        - _R_CRAN_CHECK_INCOMING_=false

(currently out of town, will say more later)

**Comment by jimhester** [comment]
yep I know how to override it on my builds, just thought we might want to change the default behavior.
